godot-core/src/builtin/transform2d.rs

```diff
@@ -150,6 +150,18 @@ impl Transform2D {
         self.glam(|aff| aff.inverse())
     }
 
+    /// Returns the determinant of the basis matrix. If the basis is uniformly
+    /// scaled, then its determinant equals the square of the scale factor.
+    /// 
+    /// A negative determinant means the basis was flipped, so one part of the
+    /// scale is negative. A zero determinant means the basis isn't invertible,
+    /// and is usually considered invalid.
+    ///
+    /// _Godot equivalent: `Transform2D.determinant()`_
+    pub fn determinant(&self) -> real {
```

Comment:
The 2nd part of the first sentence could be in the body (not header) of the docs, as it's not vital information.

You could also make the lines a bit wider, see also [here](https://godot-rust.github.io/book/contribute/conventions.html#formatting)  :slightly_smiling_face:

Reply:
Hm, not sure I understand. You mean the second sentence? The first sentence is simply:

Returns the determinant of the basis matrix.

Should I leave this on it's own line? I tried to put this at the head (not sure what the header and the body is for docs, sorry)

Reply:
Sorry, I meant first paragraph. It's also not that important, don't spend too much time on this 🙂 

Typically, the structure is:
```
/// Describes what the function does (ideally more than the function name).
///
/// Describes detailed behavior and edge cases.
/// Can be longer text.
///
/// Even multiple paragraphs.
/// 
/// # Panics
/// Conditions that lead to panic (only if applicable -- not here).
/// 
/// # Safety
/// If the function is unsafe, what to consider (only if applicable -- not here).